test/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "node/no-extraneous-import": "off",
+    "node/no-missing-import": "off"
+  }
+}
```

Comment:
These were the lint rules breaking for the test file. These seem like we want to still check for them in the main codebase? Hence the separate config